internal/pkg/plugin/plugin_installer.go

```diff
@@ -0,0 +1,50 @@
+package plugin
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/hashicorp/go-version"
+
+	"github.com/confluentinc/cli/internal/pkg/errors"
+)
+
+const (
+	programNotFoundErrorMsg      = "unable to find %s; check that it is installed in a directory in your $PATH"
```

Comment:
The second half of the error should be part of a suggestion, not the error text.

Reply:
Good point; I forgot to separate this into an error and suggestion when I switched it from a warning to an error.